gslib/commands/rm.py

```diff
@@ -56,6 +56,10 @@
 
 
 <B>DESCRIPTION</B>
+  NOTE: As part of verifying the existence of objects prior to deletion,
+  ``gsutil rm`` makes ``GET`` requests to Cloud Storage for object metadata.
```

Comment:
This is true for cp, rsync, mv as well.

Reply:
Is that to say that these other methods make two GET requests? One GET to verify an object's existence and a second GET to retrieve the object data?

If so, I'll think about if/how to add that info to those commands as well. Merging this PR in the meantime.